windows/table.cpp

```diff
@@ -130,7 +119,7 @@ void uiTableColumnAppendProgressBarPart(uiTableColumn *c, int modelColumn, int e
 
 void uiTableColumnPartSetEditable(uiTableColumn *c, int part, int editable)
 {
-	// TODO

```

Comment:
Keep this as a TODO for now, so that it shows up in my occasional `grep TODO`s and thus isn't forgotten.